packages/traw/src/state/TrawApp.ts

```diff
@@ -117,14 +129,27 @@ export class TrawApp {
       },
       user,
       document,
-      records,
+      records: {},
     };
     this.store = createVanilla(() => this._state);
+
+    this.useStore = create(this.store);
   }
 
   registerApp(app: TldrawApp) {
     app.callbacks = {
       onCommand: this.recordCommand,
+      onPatch: (app, patch, reason) => {
```

Comment:
```suggestion
      onAssetCreate: this.handleAssetCreate,
      onPatch: (app, patch, reason) => {
```

It looks like I made a mistake while merging before. Please add onAssetCreate handler too.